src/panel/details.tsx

```diff
@@ -36,47 +36,47 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
         });
     }
     render() {
-        const renderRuleDesc = (desc?: { text: string, markdown?: string }) => {
+        const renderRuleDesc = (result: Result, desc?: { text: string, markdown?: string }) => {
             if (!desc) return '—';
             return desc.markdown
-                ? <ReactMarkdown className="svMarkDown" source={desc.markdown} />
-                : desc.text;
+                ? <ReactMarkdown className="svMarkDown" source={desc.markdown} escapeHtml={false} />
```

Comment:
What trigged the addition of `escapeHtml`?

Reply:
It was for `svDetailsMessage` but not for `renderRuleDesc`

Reply:
Great!